### PR TITLE
Link to accessibility notice

### PR DESCRIPTION
**Pull request by navya9singh**
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1568069/

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
We need to not deploy this until the landing page https://www.microsoft.com/it-it/accessibility/declarations is ready

**Review by RyanCavanaugh — APPROVED** [review]

File: packages/typescriptlang-org/src/components/layout/SiteFooter.scss [hunk 1]
```diff
@@ -124,14 +131,15 @@
       }
     }
 
-    article > ul {
+    article>ul {
```
Comment (DanielRosenwasser) [inline packages/typescriptlang-org/src/components/layout/SiteFooter.scss:134]: What formatted these differently?

**Comment by DanielRosenwasser** [comment]
Is there a way to preview this? If not, can you provide a screenshot of what this looks like in both mobile and desktop?

File: packages/typescriptlang-org/src/components/layout/SiteFooter.scss (hunk 1 above)
Comment (navya9singh) [reply packages/typescriptlang-org/src/components/layout/SiteFooter.scss:134]: Just adding something new and then saving it creates these changes. I tried to remove them but saving the file again after removing something creates it.

**Comment by navya9singh** [comment]
@DanielRosenwasser  This is the desktop view 
[](url)
website1
This is the mobile view 
image

**Review by DanielRosenwasser — APPROVED** [review]

**Pull request merged**